x-pack/test/security_solution_endpoint/page_objects/index.ts

```diff
@@ -7,28 +7,28 @@
 
 import { pageObjects as xpackFunctionalPageObjects } from '../../functional/page_objects';
 import { EndpointPageProvider } from './endpoint_page';
-import { EndpointPolicyPageProvider } from './policy_page';
-import { TrustedAppsPageProvider } from './trusted_apps_page';
 import { EndpointPageUtils } from './page_utils';
 import { IngestManagerCreatePackagePolicy } from './ingest_manager_create_package_policy_page';
-import { FleetIntegrations } from './fleet_integrations_page';
 import { DetectionsPageObject } from '../../security_solution_ftr/page_objects/detections';
 import { HostsPageObject } from '../../security_solution_ftr/page_objects/hosts';
-import { ArtifactEntriesListPageProvider } from './artifact_entries_list_page';
 import { EndpointResponderPageObjects } from './endpoint_responder';
 import { TimelinePageObject } from '../../security_solution_ftr/page_objects/timeline';
+import { EndpointPolicyPageProvider } from './policy_page';
+import { TrustedAppsPageProvider } from './trusted_apps_page';
+import { FleetIntegrations } from './fleet_integrations_page';
+import { ArtifactEntriesListPageProvider } from './artifact_entries_list_page';
 
 export const pageObjects = {
   ...xpackFunctionalPageObjects,
   endpoint: EndpointPageProvider,
-  policy: EndpointPolicyPageProvider,
-  trustedApps: TrustedAppsPageProvider,
-  artifactEntriesList: ArtifactEntriesListPageProvider,
   endpointPageUtils: EndpointPageUtils,
   ingestManagerCreatePackagePolicy: IngestManagerCreatePackagePolicy,
-  fleetIntegrations: FleetIntegrations,
   detections: DetectionsPageObject,
   timeline: TimelinePageObject,
   hosts: HostsPageObject,
   responder: EndpointResponderPageObjects,
+  policy: EndpointPolicyPageProvider,
+  trustedApps: TrustedAppsPageProvider,
+  artifactEntriesList: ArtifactEntriesListPageProvider,
+  fleetIntegrations: FleetIntegrations,
```

Comment:
out of curiosity, why is the order of these fields changed?

Reply:
No reason, I moved it first in a separated file (first commit: https://github.com/elastic/kibana/pull/159014/commits/642ea23cc10277cf8ce7d12ffec8404b88f60fea) and then moved back to the same file: https://github.com/elastic/kibana/pull/159014/commits/eea7dbd05bf884eaca30d9343f27495ae621ca0e.
Those are ordered now following the different configs (endpoint first and integrations after)